cmake/Package.cmake

```diff
@@ -2,7 +2,7 @@ if(QT_MOC_EXECUTABLE)
   get_filename_component(_qt_bin_dir ${QT_MOC_EXECUTABLE} DIRECTORY)
 endif()
 find_program(LINUXDEPLOY_EXECUTABLE linuxdeploy linuxdeploy-x86_64.AppImage)
-find_program(MACDEPLOYQT_EXECUTABLE macdeployqt HINTS "${VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}/tools/Qt6/bin" NO_DEFAULT_PATH)
+find_program(MACDEPLOYQT_EXECUTABLE macdeployqt HINTS "${VCPKG_INSTALLED_DIR}/arm64-osx/tools/Qt6/bin" NO_DEFAULT_PATH)
```

Comment:
👀

Reply:
@m-kuhn , not sure if it's caused by updating vcpkg or Qt, but on the x64-osx, we're building both x64 and arm64, the latter is treat as host (like iOS).

Reply:
I think there should be something like this in `QT_HOST_PATH`. I'd keep it at least as a comment that we took the shortcut here.

Reply:
+1 to comment